parts/k8s/addons/kubernetesmasteraddons-nvidia-device-plugin-daemonset.yaml

```diff
@@ -21,6 +21,10 @@ spec:
       # This, along with the annotation above marks this pod as a critical add-on.
       - key: CriticalAddonsOnly
         operator: Exists
+      - key: nvidia.com/gpu
+        effect: NoSchedule
+        operator: Equal
+        value: "true"
```

Comment:
Since you are adding the toleration, can you also replace the snippet from below at `/etc/docker/daemon.json"` in `kubernetesagentcustomdata.yml` since we want it to be enabled for GPU pools only but not for CPU pools.

```
      }{{if IsNSeriesSKU .}}{{if IsNVIDIADevicePluginEnabled}}
      ,"default-runtime": "nvidia",
      "runtimes": {
         "nvidia": {
             "path": "/usr/bin/nvidia-container-runtime",
             "runtimeArgs": []
        }
      }{{end}}{{end}} 
```

Reply:
Fixed